mindmeld/converter/dialogflow.py

```diff
@@ -379,26 +386,15 @@ def create_mindmeld_init(self):
 
                                 replies = data if isinstance(data, list) else [data]
 
-                                if datastore["fallbackIntent"]:
-                                    function_name = "default" + "_" + language
-                                    if language == "en":
-                                        # TODO: support multiple defaults for languages
-                                        handles = [
-                                            "default=True",
-                                            "intent='unsupported'",
-                                        ]
-                                    else:
-                                        handles = ["intent='unsupported'"]
-                                else:
-                                    function_name = "renameMe" + str(i) + "_" + language
-                                    handles = [
-                                        "intent="
-                                        + "'"
-                                        + self.clean_name(datastore["name"])
-                                        + "_usersays_"
-                                        + language
-                                        + "'"
-                                    ]
+                                function_name = "renameMe" + str(i) + "_" + language
+                                handles = [
+                                    "intent="
+                                    + "'"
+                                    + self.clean_name(datastore["name"])
+                                    + "_"
+                                    + language
+                                    + "'"
+                                ]
```

Comment:
f-string please

Reply:
done

Reply:
i don't think we can use fstring here since it is only available on python 3.6 and above while mindmeld still supports py3.5